pkg/provisioner/ironic/ironic.go

```diff
@@ -1347,6 +1385,8 @@ func (p *ironicProvisioner) Provision(hostConf provisioner.HostConfigData) (resu
 		return transientError(provisioner.NeedsRegistration)
 	}
 
+	_ = p.updateAutomatedClean(ironicNode)
```

Comment:
If the error was a 409, we need to pause before trying anything else because the host is busy. If it's not a 409, we probably don't want to ignore the error.

Reply:
Fixed it. Thanks.